ElectronicObserver/Utility/Data/EquipmentFitBonus.cs

```diff
@@ -1,16 +1,51 @@
-﻿using System.Collections.Generic;
-using ElectronicObserverTypes;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
-using ElectronicObserverTypes.Serialization.FitBonus;
+using ElectronicObserverTypes;
 using ElectronicObserverTypes.Extensions;
+using ElectronicObserverTypes.Serialization.FitBonus;
 
 namespace ElectronicObserver.Utility.Data;
 
 public static class EquipmentFitBonus
 {
+	public static FitBonusValue GetFitBonus(this IShipData ship)
+	{
+		if (!ship.MasterShip.ASW.IsDetermined) return new();
+		if (!ship.MasterShip.Evasion.IsDetermined) return new();
+		if (!ship.MasterShip.LOS.IsDetermined) return new();
```

Comment:
An improvement here could be to still check the fit bonus even if these values aren't determined. For example, if only asw wasn't determined, you could still check all other values and only set asw to 0.

Reply:
Good idea, but I shouldn't send data to api if those values are not determined

Reply:
Maybe just include boolean properties that tell you if those values are determined. I don't think just ignoring the incorrect fit because those values aren't determined makes sense.